docker_templates/templates/docker_images/ros2_source_entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+# setup ros2 environment
+echo "export ROS_SECURITY_ROOT_DIRECTORY=$ROS2_WS/demo_keys" >> ~/.bashrc
+echo "export ROS_SECURITY_ENABLE=true" >> ~/.bashrc
+echo "export ROS_SECURITY_STRATEGY=Enforce" >> ~/.bashrc
+echo "export RMW_IMPLEMENTATION=rmw_fastrtps_cpp" >> ~/.bashrc
```

Comment:
All these were for the sake of the sros2 demo. I'm not sure we should set them, or if we do maybe with different values because these will result in security exceptions (if `$ROS2_WS/demo_keys` doesnt exist with valid subfolders, keys and certificates for each node the user will launch). So I'm afraid that it'd lead to bad user experience if talker/listener doesnt work out of the box.
The alternatives I see are:
- not set any of these environment variables:
  - if RMW_IMPLEMENTATION is not defined it will default to fastrtps anyway
  - the user can define the security environment variables as described in the tutorial that he'll have to follow to create keys and certs anyway
- Change the values of these env variables:
  - not set `ROS_SECURITY_ROOT_DIRECTORY` letting the user decide where he will store the files when creating them
  - disable security by default so that unsecure nodes can talke to each other by default (`export ROS_SECURITY_ENABLE=false`)